free-programming-books.md

```diff
@@ -315,6 +315,7 @@
 * [Algorithms and Automatic Computing Machines (1963)](https://archive.org/details/Algorithms_And_Automatic_Computing_Machines) - B. A. Trakhtenbrot
 * [Algorithms and Complexity](https://www.math.upenn.edu/~wilf/AlgoComp.pdf) - Herbert S. Wilf (PDF)
 * [Algorithms Course Materials](http://jeffe.cs.illinois.edu/teaching/algorithms/) - Jeff Erickson
+* [Algorithms Notes for Professionals](https://goalkicker.com/AlgorithmsBook) (PDF)
```

Comment:
For goalkicker books, please add a contributor not "Compiled from StackOverflow documentation"

Reply:
I have added it in the second commit I made in the same PR.

Thank you